client/src/components/embedding/index.tsx

```diff
@@ -81,16 +78,6 @@ const Embedding = (props: Props) => {
     await dispatch(
       actions.layoutChoiceAction(e.currentTarget.value, isSidePanel)
     );
-    if (
-      imageUnderlay &&
-      !isSidePanel &&
-      !e.currentTarget.value.includes(globals.spatialEmbeddingKeyword)
-    ) {
-      dispatch({
```

Comment:
this is not needed anymore, since the toggle state should only be controlled by the image toggle button only